experimental/devices/ads1x15/ads1x15_test.go

```diff
@@ -120,13 +120,12 @@ func TestPinADC_Read(t *testing.T) {
 	}
 }
 
-/* https://github.com/google/periph/issues/334
 func TestPinADC_ReadContinous(t *testing.T) {
 	b := i2ctest.Playback{
```

Comment:
Since it's an internal race condition, please set DontPanic to true.
https://periph.io/x/periph/conn/i2c/i2ctest#Playback
This way, the potentially extra data will be ignored.

Reply:
Thanks!